multihost_test/Bugzillas/test_bz_automation.py

```diff
@@ -270,3 +270,95 @@ def test_bz_955769(self, multihost):
         cmd1 = execute_cmd(multihost, "ls -Zal /home/testBZ955769").stdout_text
         assert cmd1 == cmd
         execute_cmd(multihost, "userdel -rfZ testBZ955769")
+
+    @pytest.mark.tier1
+    def test_bz_951743(self, multihost):
+        """
+        :title: BZ#951743 ((usermod -U) can't undo (passwd -l))
```

Comment:
I had to review the meaning of the options to understand what the test did. I propose to change the title, or add a description to improve this situation. Example:
Unlock a user's password with "usermod -U" after the user's password has been locked with "passwd -l".